### DISCUSSION TITLE
Webpack plugin causes issues with typescript loader

### DISCUSSION THREAD

**Discussion by lukasbash** [microsoft/monaco-editor#3656] [Q&A]
Using:
```
node: v8.16.1
npm: 6.4.1
monaco-editor: 0.17.1
monaco-editor-webpack-plugin: 1.7.0 
react-monaco-editor: 0.30.1
webpack: 3.12.0
```

As soon as I add the webpack plugin to my webpack configuration, it seems that the typescript loader does not work anymore. With the plugin I get the following issue:

```
ERROR in ./src/index.tsx
Module parse failed: Unexpected token
You may need an appropriate loader to handle this file type.
| function doSomething(): boolean
```

It seems like it does not reach the typescript loader anymore.

------

Offtopic: Is this module actually maintained? I see no recent responses on issues and no further commits on this repository.

**Comment by alexdima** [comment]
This sample works fine for me -- https://github.com/microsoft/monaco-editor-samples/tree/master/browser-esm-webpack-typescript

So it appears that this plugin can, at least in toy examples, coexist with `ts-loader`.

Could you maybe try starting from that sample and slowly changing your webpack config towards your current one?
